setup.py

```diff
@@ -75,11 +75,13 @@ def package_files(directory):
 logging.debug("{} env var is set: {}".format(_MLFLOW_SKINNY_ENV_VAR, _is_mlflow_skinny))
 
 setup(
-    name="mlflow",
-    version=version,
+    name="mlflow" if not _is_mlflow_skinny else "mlflow-skinny",
+    version=version if not _is_mlflow_skinny else "0.1.0",  # Will be version after a release
```

Comment:
@eedeleon Do we need this conditional? Ideally, we'll use version 1.14.0 for both the core and skinny client the next time we have a release.

Reply:
I can remove it